internal/pkg/service/common/dependencies/mocked.go

```diff
@@ -38,49 +38,66 @@ type mocked struct {
 }
 
 type MockedValues struct {
-	Services                           storageapi.Services
-	Features                           storageapi.Features
-	Components                         storageapi.Components
-	StorageAPIHost                     string
-	StorageAPIToken                    storageapi.Token
-	StorageAPITokenMockedResponseTimes int
+	services                           storageapi.Services
+	features                           storageapi.Features
+	components                         storageapi.Components
+	storageAPIHost                     string
+	storageAPIToken                    storageapi.Token
+	storageAPITokenMockedResponseTimes int
+
+	useRealAPIs         bool
+	storageAPIClient    client.Client
+	encryptionAPIClient client.Client
+	schedulerAPIClient  client.Client
 }
 
 type MockedOption func(values *MockedValues)
 
+func WithTestProject(project *testproject.Project) MockedOption {
```

Comment:
Tu sa vytvaraju `MockedDeps`, vsetko mame cez options, ale bol tu jeden setter `SetFromTestProject` (vid nizsie).
Teda prerobil som ho na option, nech to nemyli.